modules/op-release-notes-1-0.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assembly:
+//
+// * openshift_pipelines/op-release-notes.adoc
+
+[id="op-release-notes-1-0_{context}"]
+= Release Notes for Red Hat {pipelines-title} Technology Preview 1.0
+
+[id="new-features-1-0_{context}"]
+== New features
+{pipelines-title} Technology Preview (TP) 1.0 is now available on {product-title} 4.4. {pipelines-title} TP 1.0 is updated to support:
+
+* Tekton Pipelines 0.11.3
+* Tekton `tkn` CLI 0.9.0
+* Tekton Triggers 0.4.0
+* ClusterTasks based on Tekton Catalog 0.11
+
+In addition to the fixes and stability improvements, here is a highlight of what’s new in OpenShift Pipelines 1.0.
+
+=== Pipelines
+* Support for  v1beta1 API Version.
+* Support for an improved LimitRange. Previously, LimitRange was specified exclusively for the TaskRun and the PipelineRun. Now there is no need to explicitly specify the LimitRange. The minimum LimitRange across the namespace is used.
+* Support for sharing data between Tasks using TaskResults and TaskParams.
+* Pipelines can now be configured to not overwrite the `HOME` environment variable and `workingDir` of Steps.
+* Similar to Task Steps, `sidecars` now support script mode.
+* You can now specify a different scheduler name in TaskRun `podTemplate`.
+* Support for variable substitution using Star Array Notation
+* Tekton Controller can now be configured to monitor an individual namespace.
+* A new description field is now added to the specification of Pipeline, Task, ClusterTask, Resource, and Condition.
+* Addition of proxy parameters to Git PipelineResources.
+
+=== Pipelines CLI
+
+* The `describe` sub-command is now added for the following `tkn` resources -- `eventlistener`, `condition`, `triggertemplate`, `clustertask`, and `triggerbinding`.
+* Support added for `v1beta1` to the following commands along with backward comptibility for `v1alpha1` - `clustertask`, `task`, `pipeline`, `pipelinerun`,  and `taskrun`.
+* The following commands can now list output from all namespaces using the `--all-namespaces` flag option:
+** `tkn task list`
+** `tkn pipeline list`
+** `tkn taskrun list`
+** `tkn pipelinerun list`
+
+The output of these commands is also enhanced to display information without headers using the `--no-headers` flag option.
+
+* You can now start a Pipeline using default parameter values by specifying `--use-param-defaults` flag in the `tkn pipelines start` command.
+* Support for Workspace is now added to `tkn pipeline start` and `tkn task start` commands.
+* A new `clustertriggerbinding` command is now added with sub-commands - `describe`, `delete`, and `list`.
```

Comment:
either get rid of the hyphen, or change to "now added with the following subcommands:"